hw/main2.py

```diff
@@ -0,0 +1,33 @@
+# 2. Напишите функцию группового переименования файлов. Она должна:
+# * принимать в качестве аргумента желаемое конечное имя файлов. 
+# * При переименовании в конце имени добавляется порядковый номер.
+# * принимать в качестве аргумента расширение исходного файла. 
+# Переименование должно работать только для этих файлов внутри каталога.
+# * принимать в качестве аргумента расширение конечного файла.
+
+
+import os
+
+def rename_files(directory, desired_name, original_extension, final_extension):
+    if not os.path.isdir(directory):
+        print(f"Directory '{directory}' does not exist.")
+        return
```

Comment:
"Явное лучше неявного". Если хотите так выйти из функций, тогда уж пишите конкретный возврат.
```suggestion
        return None
```